### ISSUE TITLE
Can't find .exe PLEASE HELP!!!!!!

### ISSUE THREAD

**Issue by Achtman** [Maxx53/NoCableLauncher#3]
The cable works
The game works
The speakers work
The guitar works
The patch half works

I bought the steam version of Rocksmith2014 on steam and I am using a third party USB to guitar cable.

This patch worked perfectly when I un-checked the steam game box, but when the steam game box is checked it brings up an error "Can't find process: Rocksmith2014.exe"

I want to run the game through steam but it keeps giving me that error  please help!!!!

**Comment by Maxx53** [comment]
Seems your game starting too long time. try to increase **waitTime** parameter in "settings.xml" file. It's time that launcher wait while game starts.

**Comment by Achtman** [comment]
Thank you so much changing the wait time worked!!!

**Comment by jitsumoreno** [comment]
@achtman could you please document here your OS and what cable you used? (Amazon link would be awesome).

Is the fix still working for you? Do you have comments worth sharing?

I believe this to be really useful information so that others can replicate your sucess. 

Thanks and sorry for commenting on a closed thread.

**Comment by Achtman** [comment]
@jitsumoreno yes the fix is still working almost perfectly! 

Here is the cable I use.
https://www.amazon.com/gp/aw/d/B0132WP93U/ref=mp_s_a_1_102?ie=UTF8&qid=1471475533&sr=8-102&pi=AC_SX236_SY340_QL65&keywords=usb+to+guitar&dpPl=1&dpID=51PrClM0dkL&ref=plSrch

I currently use rocksmith  2014 on some older HP laptop running windows 10 (pavilion g7 I think?).

I tried dozens of different patches before this one, and this one is the only one that I could get to work. 

My original problem I had was that rocksmith 2014.exe  was starting up too slow so that the patch couldn't find the  .exe running. Rocksmith would still launch, but not with the help of the patch so that rocksmith didn't recognize the cable. 
All I had to do was increase the time that the patch waited to find the  .exe, and now everything works just fine. 

I also had to disable all playback/microphone sound devices except for the USB cable. 

The only problems I've run into are caused by the cheap cable I bought. Once in a great while the notes will come out extremely delayed but this can be easily fixed by restarting the game. 

Overall I would definitely recommend buying a cheaper cable and using this patch, but maybe a little higher quality than the one I got. Hope I helped someone :)
